Assets/Scripts/Dialogue/DialogueBubble.cs

```diff
@@ -35,52 +39,45 @@ private struct VertexAnim
 
     void Awake()
     {
+        RectTrans = GetComponent<RectTransform>();
+        Arrow = transform.parent.gameObject.transform.GetChild(1).gameObject;
+        ArrowAnim = transform.parent.GetComponentInChildren<Animator>();
         Text = GetComponentInChildren<TMP_Text>();
-        ActiveEffect = TextEffect.None;
         // Store original speed to restore after acceleration
         OriginalTextSpeed = TextSpeed;
-    }
-
-    void Start()
-    {
         Group = GetComponent<CanvasGroup>();
         Group.alpha = 0;
     }
 
     void Update()
     {
-        if (Input.GetKeyDown(KeyCode.J))
-        {
-            IsTextShaking = !IsTextShaking;
-        }
-
         // Simple hacky way to accelerate the text speed.
-        if (Input.GetButtonDown("Fast Forward")) {
-            TextSpeed = 100;
-        }
-
-        // Temporary for showcase purposes
-        if (IsTextShaking)
+        if (Input.GetButtonDown("Fast Forward"))
         {
-            StartCoroutine(ShakeText());
+            TextSpeed = FastForwardTextSpeed;
         }
+
+        // Update arrow position
+        Arrow.transform.position = new Vector3(transform.position.x + RectTrans.rect.width / WidthOffsetScaler, transform.position.y, 0);
```

Comment:
What is this used for? :3 I'm only looking through the code right now so I can't really tell 😅 Is it an animation or an adjustment of some kind?

Reply:
I added an animation to the dialogue box, this is to place it in the correct position because the box changes height dynamically